docker/volumeplugin/rancher.go

```diff
@@ -17,6 +17,7 @@ const (
 )
 
 var goodStates = map[string]bool{
+	"inactive":     true,
```

Comment:
Volume fails to mount. When you've just created the volume in the UI it actually doesn't get created for real: it's only in Rancher's mind. 

When you start the first container using the volume, it gets created and then mounted. So in this mount step, if `inactive` is not in this list, the driver just won't find it, because it's in inactive state in Rancher.